### DISCUSSION TITLE
problems sending messages in whatsapp cloud api

### DISCUSSION THREAD

**Discussion by maynorgc21** [chatwoot/chatwoot#5343] [Self hosted] [answered]
Hi, my whatsapp cloud api messages are not being sent but I can receive messages.

They work only the first day, the second day no longer work, and even try to reinstall chatwoot and is the same mechanism, it works only the first day but the second is no longer sent.

Could you please help me or guide me a little to what I'm doing wrong, thank you very much.

**Comment by pucilpet** [comment, marked as answer]
Are you using test credentials for whatsapp? Facebook provides test credentials which are only valid for 24 hours.

See here how to setup live credentials: https://www.chatwoot.com/docs/product/channels/whatsapp/whatsapp-cloud
